.gitignore

```diff
@@ -21,5 +21,6 @@ project.lock.json
 .secrets
 appsettings.json
 local.settings.json
+response.json
```

Comment:
I considered adding to gitignore but started using `| tee ../response.json` and `jq -r '.statusQueryGetUri' ../response.json` as an alternative.

No preference either way but an update to the readme would be required if you decide it is preferable.

Reply:
Yes but in the parent directory so doesn't flagged in git.

Reply:
Creating the fie within the repo's dir structure is going to avoid unforeseen problems.